supybot_fedora/plugin.py

```diff
@@ -721,11 +767,12 @@ def swedish(self, irc, msg, args):
         Humor mmcgrath."""
 
         # Import this here to avoid a circular import problem.
-        from __init__ import __version__
+        from .__init__ import __version__
```

Comment:
It would be `from . import __version__`, it's a bit unusual to import from `__init__`

Reply:
fixed.